src/components/qas/questions/qaCalculators/distanceCalculator/index.jsx

```diff
@@ -0,0 +1,279 @@
+      const ly = parsecs * 3.26;
+      return this.roundTo(ly, 10);
+    }
+    return '?';
+  };
+
+  solveForMegaLightYears = lightYears => {
+    if (lightYears !== '?') {
+      const mly = lightYears / 999315.5373;
+      return this.roundTo(mly, 10);
+    }
+    return '?';
+  };
+
+  roundTo = (number, decimalPlace) => {
+    const roundedNumber = Math.round(number * decimalPlace) / decimalPlace;
+    return roundedNumber;
+  };
+
+  addTheCommas = num =>
+    num.toString().search(/e/gi) < 0
+      ? num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+      : num;
+
+  render() {
+    const { question, answer, activeId } = this.props;
+    const {
+      hasFocus,
+      answerable,
+      peakMagnitude,
+      distanceModulus,
+      parsecs,
+      megaParsecs,
+      lightYears,
+      megaLightYears,
+    } = this.state;
+    const { id, questionType, label, placeholder } = question;
+    const isTextArea = questionType === 'textArea';
+    const active = activeId === id;
+    const answered = !isEmpty(answer);
+    const cardClasses = classnames('qa-card', { active: hasFocus });
+    const fieldClasses = classnames('qa-text-input', {
+      answered,
+      unanswered: !answered,
+      answerable: answerable || answered || active,
+    });
+
+    return (
+      <Card className={cardClasses}>
+        <CardText>
+          <TextField
+            ref={this.field}
+            id={`text-${isTextArea ? 'area' : 'input'}-${id}`}
+            className={fieldClasses}
+            type="number"
+            min="0"
+            label={<div dangerouslySetInnerHTML={renderDef(label)} />}
+            lineDirection="center"
+            placeholder={placeholder}
+            defaultValue={answered ? answer.content : null}
+            onBlur={this.handleBlur}
+            onFocus={this.handleFocus}
+            onChange={this.handleChange}
+            disabled={!(answerable || answered || active)}
+          />
+        </CardText>
+        <CardActions expander>
+          <p>View Equation and Answers</p>
+        </CardActions>
+        <CardText expandable>
+          <div>
+            <p>DM = {this.addTheCommas(peakMagnitude)} + 19.4</p>
+          </div>
+          <br />
+          <div>
+            <p>
+              d = 10{' '}
+              <span className="exponential-fraction">
+                <span className="numerator">
+                  ({this.addTheCommas(distanceModulus)} + 5)
+                </span>
+                <span className="denominator">/ 5</span>
+              </span>
+            </p>
+          </div>
+          <br />
+          <div>
+            <div>
+              Parsecs (pc) ={' '}
+              <TextField
+                id="parsecs"
+                className="calculated-measurement"
+                fullWidth={false}
+                size={40}
+                disabled
+                type="text"
+                value={this.addTheCommas(parsecs)}
+              />
```

Comment:
I agree with you and would much rather have them as inline block elements. I'd be happy to rebuild them since I can't really see a use case immediately.